src/org/zaproxy/zap/extension/authenticationstatus/AuthenticationChecklistPanel.java

```diff
@@ -0,0 +1,217 @@
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/tick-circle.png"));
+		CONFIGURATION_NOT_VALIDATED_YET_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/question-white.png"));
+		BAD_CONFIGURATION_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/cross-circle.png"));
+		MINIMUM_CONFIGURATION_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/exclamation-circle.png"));;
+	}
+
+	public AuthenticationChecklistPanel() {
+		contextStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.context"));
+		userStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.user"));
+		authenticationMethodStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.authentication"),
+				JLabel.LEFT);
+		loggedInOutIndicatorStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.loggedInOutIndicator"),
+				JLabel.LEFT);
+
+		contextStatusLabel.setIcon(CONFIGURATION_NOT_VALIDATED_YET_ICON);
+		userStatusLabel.setIcon(CONFIGURATION_NOT_VALIDATED_YET_ICON);
+		authenticationMethodStatusLabel.setIcon(CONFIGURATION_NOT_VALIDATED_YET_ICON);
+		loggedInOutIndicatorStatusLabel.setIcon(CONFIGURATION_NOT_VALIDATED_YET_ICON);
+
+		setLayout(new GridBagLayout());
+
+		int y = 0;
+		add(contextStatusLabel, LayoutHelper.getGBC(0, y, 2, 0D));
+		add(authenticationMethodStatusLabel, LayoutHelper.getGBC(0, ++y, 2, 0D));
+		add(loggedInOutIndicatorStatusLabel, LayoutHelper.getGBC(0, ++y, 2, 0D));
+		add(userStatusLabel, LayoutHelper.getGBC(0, ++y, 2, 0D));
+
+		Border innerBorder = BorderFactory
+				.createTitledBorder(Constant.messages.getString("authenticationstatus.dialog.verification.title"));
+		Border outerBorder = BorderFactory.createEmptyBorder(5, 5, 5, 5);
+		setBorder(BorderFactory.createCompoundBorder(outerBorder, innerBorder));
+
+	}
+
+	public boolean configuredSuccessfully() {
+		return this.configuredSuccessfully;
+	}
+
+	protected void updateConfigurationStatus(Context context) {
+		if (context == null) {
+			setBadContextStatus();
+			logger.debug("authentication configuration: bad - context not defined");
+			return;
+		}
+		setGoodContextStatus();
+
+		this.updateAuthenticationMethodStatus(context.getAuthenticationMethod());
+	}
+
+	private void updateAuthenticationMethodStatus(AuthenticationMethod authenticationMethod) {
+		if (authenticationMethod == null) {
+			this.setBadAuthenticationMethodStatus();
+			this.setBadIndicatorStatus();
+			logger.debug("authentication configuration: bad - authentication method not defined");
```

Comment:
updating the relevant label text with this kind of message will be useful, right?

Reply:
Yes, or using a tool tip.